portfolio/src/main/java/com/google/sps/servlets/CommentServlet.java

```diff
@@ -22,38 +23,43 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.SortDirection;
 
 /** This servet is responsible for the comment data **/
 @WebServlet("/commentServlet")
 public class CommentServlet extends HttpServlet {
+  private static final String USERNAME_PARAMETER = "username";
+  private static final String REFLECTION_PARAMETER = "reflection";
+  private static final String COMMENT_PARAMETER = "Comment";
+  private static final String TIMESTAMP_PARAMETER = "timestamp";
+  
+  private String USERNAME_INPUT;
+  private String REFLECTION_INPUT;
+  private long ID;
+  private long TIMESTAMP_OF_COMMENT;  
```

Comment:
These variable names are not `private static final` so they should be formatted like `userNameInput`, `reflectionInput`, etc...